src/cpp/session/SessionOptions.cpp

```diff
@@ -501,7 +501,10 @@ namespace {
 FilePath macBinaryPath(const FilePath& resourcePath,
                        const std::string& stem)
 {
-   // first check for Electron binary path
+   // otherwise, look in default Qt location
+   FilePath qtPath = resourcePath.getParent().completePath("MacOS").completePath(stem);
+   return qtPath;
```

Comment:
Does this code need a Qt-only (or existence) check? It seems like the following Electron code would now be dead code?